packages/contentstack-utilities/src/contentstack-management-sdk.ts

```diff
@@ -43,32 +51,32 @@ export default async (config) => {
         },
       },
       refreshToken: () => {
-        return new Promise((resolve, reject) => {
-          reject('You do not have permissions to perform this action, please login to proceed');
-        });
+        return Promise.reject('You do not have permissions to perform this action, please login to proceed');
       },
     };
+
     if (typeof config.branchName === 'string') {
-      option['headers'] = {
-        branch: config.branchName,
-      };
+      option.headers.branch = config.branchName;
+    }
+
+    if (this.analyticsInfo) {
+      option.headers['X-CS-CLI-ID'] = config.analyticsInfo;
```

Comment:
@shafeeqd959 Are we using `X-CS-CLI` OR `X-CS-CLI-ID`?

Reply:
> @shafeeqd959 Can you please add the ticket number and version update information in the description of this PR?

Sure

Reply:
> @shafeeqd959 Are we using `X-CS-CLI` OR `X-CS-CLI-ID`?
I will update this to  `X-CS-CLI`